src/view/components/common/main-navigation/main-navigation.ts

```diff
@@ -22,13 +25,24 @@ export class MainNavigation {
         routingService.menusChange$.subscribe(menu => {
             this.menuItems = menu;
         });
+
+        routingService.currentPortlet$.subscribe(id => {
+            this.open = !this.open;
+            this.menuItemIdActive = id;
+            this.menuActiveTabName = this.getMenuSelected(id).tabName;
+        });
     }
 
     /**
      * Change or refresh the portlets from the main menu
-     * @param link portlet url
+     * @param menuItem portlet url
      */
-    public gotToPage(link: string): void {
+    public gotToPage(menuItem: MenuItem): void {
```

Comment:
why did you change the firm?. At the end you are only taking the menulink, I dont get why do you pass the whole menu item